api/users/permissions.py

```diff
@@ -14,6 +14,13 @@ def has_object_permission(self, request, view, obj):
         else:
             return obj == request_user
 
+class CurrentUser(permissions.BasePermission):
+    """ Check to see if the request is coming from the currently logged user
+    """
+
+    def has_object_permission(self, request, view, obj):
+        assert isinstance(obj, User), 'obj must be a User, got {}'.format(obj)
+        return obj == request.user
```

Comment:
I think I need to keep check objects, as chekcing ` _id `s causes this to pop up ` 'AnonymousUser' object has no attribute '_id' `, any idea why?